modules/troubleshooting/pages/registries.adoc

```diff
@@ -63,27 +63,27 @@ skopeo login --authfile /tmp/auth.json "$should_have_access_to"
 Note: works for `dockercfg` and `dockerconfigjson` secrets, not `basic-auth`.
 
 [#check-if-the-secret-is-linked-to-the-service-account]
-=== Check if the secret is linked to the `appstudio-pipeline` service account
+=== Check if the secret is linked to the `build-pipeline-$COMPONENT_NAME` service account
```

Comment:
Again, this will be a different SA if it's a release pipeline, so it should probably say "to your service account" or something like that? This comment applies to the changes below as well.

Reply:
changed it to general SERVICE_ACCOUNT_NAME